development/prow-installer/cmd/installer/main.go

```diff
@@ -48,6 +48,11 @@ func main() {
 
 	ctx := context.Background()
 
+	// this variable must be set
+	if err := os.Setenv("GOOGLE_APPLICATION_CREDENTIALS", *credentialsFile); err != nil {
```

Comment:
You should check if it is already set and print a message out. The user might not expect this behavior and after running the installer will do work on the wrong Google Project or the wrong cluster.

Reply:
We pass the credentials as a flag of an application then set the variable based on that flag. If the user does not provide the flag program exits with a fail. This should not be the case there.

Reply:
So, after some testing I can confirm that the `os.Setenv()` does not change the environment variables of a parent process. That means overriding the env variable is not the case here. The [Gist](https://gist.github.com/Ressetkk/188cdd50583c0b5f04f6c97a8fd2fa52) confirms it.